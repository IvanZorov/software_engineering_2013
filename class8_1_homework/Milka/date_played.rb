require 'csv'
require 'date'
played = Array.new()
date = Date.strptime(ARGV[1], '%d/%m/%Y')
CSV.foreach(ARGV[0]) do |csv|
	csv_date = Date.strptime(csv[0], '%d/%m/%Y')
	played << [csv[1], csv[2]] if (date == csv_date)
end
CSV.open("date_played_result.csv", "w") do |row|
	played.each do |element|
		row << element
	end
end