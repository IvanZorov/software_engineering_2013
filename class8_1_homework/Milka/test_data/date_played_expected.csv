CHI-BOS,91-45
LAC-MIA,146-126
